cli/src/template_builder.rs

```diff
@@ -436,7 +436,7 @@ fn build_timestamp_method<'a, L: TemplateLanguage<'a>>(
                 time_util::format_absolute_timestamp_with(&timestamp, &format)
             }))
         }
-        "UTC" => {
+        "utc" => {
```

Comment:
This commit should be squashed into the previous one. See https://github.com/martinvonz/jj/blob/main/docs/contributing.md#code-reviews